addons/mail/controllers/main.py

```diff
@@ -84,8 +97,8 @@ def read_subscription_data(self, res_model, res_id, follower_id=None):
         subtypes_list = sorted(subtypes_list, key=itemgetter('parent_model', 'res_model', 'internal', 'sequence'))
         return subtypes_list
 
-    @http.route('/mail/view', type='http', auth='none')
-    def mail_action_view(self, model=None, res_id=None, message_id=None):
+    @http.route('/mail/view', type='http', auth='none', methods=['GET'])
+    def mail_action_view(self, token, model=None, res_id=None, message_id=None):
```

Comment:
Oh that one. Thought you were talking about all modified routes. Indeed this one can probably be kept.